app/src/main/res/layout/fragment_rules.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/bgColor"
```

Comment:
I do not sure it is good idea every time write this attribute, try to set default background for all views in root style. If it works correct please remove this attribute in all other cases.